src/renderer/store/modules/utils.js

```diff
@@ -614,6 +623,20 @@ const mutations = {
     state.sessionSearchHistory = history
   },
 
+  setDeArrowCache (state, cache) {
+    state.deArrowCache = cache
+  },
+
+  addVideoToDeArrowCache (state, payload) {
```

Comment:
Might be better to whitelist the expected properties
I think right now we only need `videoId` & `titles`?